CHANGELOG.md

```diff
@@ -1,5 +1,91 @@
 # CHANGELOG
 
+## [1.7.0]
+
+### New features:
+
+Gateway:
+* add `GatewayClass.Spec.ParametersRef` support [#4157](https://github.com/kumahq/kuma/pull/4157) @michaelbeaumont
+* cp annotations from gateway to svc [#4327](https://github.com/kumahq/kuma/pull/4327) @johnharris85
+* only reconcile Gateway when GatewayClass is Ready [#4162](https://github.com/kumahq/kuma/pull/4162) @michaelbeaumont
+
+Observability:
+* hijack application metrics [#4286](https://github.com/kumahq/kuma/pull/4286) @lukidzi
```

Comment:
this is big, can we emphasize it a little bit? Something like
`hijack application metrics to enable scraping metrics from mTLSed applications without prometheus in the mesh`

Reply:
ack

Reply:
done